trycourier/client.py

```diff
@@ -176,3 +176,28 @@ def merge_profile(self, recipient_id, profile):
             raise CourierAPIException(resp)
 
         return resp.json()
+
+    def get_message_status(self, message_id):
```

Comment:
Hey, thanks for getting back on this. I've updated the name of the method, and added a test for the failure case.

RE `make test` -- didn't do this the first time round as I didn't spot the `Makefile`. So I used `pytest` to run tests instead. It would be great to document this somewhere (perhaps it already is and I've overlooked it). In any case, have removed the errant blank line from `client.py`!

Reply:
Sorry about that. I'll be sure to add it to the CONTRIBUTING.md. It also appears that the GitHub Action that runs the test isn't firing. I'll also look into that.